userguide/storage.rst

```diff
@@ -1606,25 +1606,24 @@ compromised. **Immediately** create a new passphrase and recovery key.
 View Multipaths
 ~~~~~~~~~~~~~~~
 
+This option is only displayed on systems that contain multipath-capable
+hardware.
+
 %brand% uses
 `gmultipath(8) <https://www.freebsd.org/cgi/man.cgi?query=gmultipath>`__
 to provide
 `multipath I/O <https://en.wikipedia.org/wiki/Multipath_I/O>`__
-support on systems containing hardware that is capable of multipath.
-An example would be a dual SAS expander backplane in the chassis or an
-external JBOD.
+support on systems containing multipath-capable hardware. For example,
```

Comment:
"support on systems containing multipath-capable hardware like
a chassis equipped with a dual SAS expander backplane, or an
external JBOD with multipath cabling."
(Check with Support for the proper way to phrase those last three words.)

Reply:
Checked with Caleb. Suggestion is: "...JBOD that is wired for multipath."